mem/buffers.go

```diff
@@ -119,20 +122,25 @@ func Copy(data []byte, pool BufferPool) Buffer {
 	return NewBuffer(buf, pool)
 }
 
+// ReadOnlyData returns the underlying byte slice of the Buffer.
```

Comment:
Similar as above.  This type is unexported; these methods should not need comments.